### ISSUE TITLE
Restart NVDA when Dism++ has been opened, Screen Review Mode will be lost.

### ISSUE THREAD

**Issue by vgjh2005** [nvaccess/nvda#7900]
### Steps to reproduce:
> 0 Insure NVDA is opened.
> 1. Open dism++.
> 2. Try to browse dism++ window with Screen Review Mode.  Successful.
> 3. Restart NVDA in the dism++ window.
### Expected behavior:
> Try again. Content in this window is still available to be browsed with Screen Review Mode.

### Actual behavior:
> I am failure.

### System configuration:
NVDA version:next-14773,a3368231 And 2017.4

NVDA Installed or portable: installed
Windows version: Windows 10 LTSB 2016 And Windows 10Ver1709 (64 Bit) build 16299.192

### Other questions:

Does the issue still occur after restarting your PC?
no
Have you tried any other versions of NVDA?
no

**Comment by man0528** [comment]
Yeah, I encountered the same Bug

**Comment by vgjh2005** [comment]
according to my research, WeChat for windows is the same.

**Comment by veg-xiaobo** [comment]
I reproduction this bug.

**Comment by Adriani90** [comment]
Is anyone still having this problem in NVDA 2019.1.1?

**Comment by lukaszgo1** [comment]
I can still reproduce this issue. Restarting Dism+ helps, but given the nature of this tool it cannot be always done.

**Comment by man0528** [comment]
Yes, I can still reproduce this behavior.

**Comment by leonardder** [comment]
Screen review works as follows:

1. An application draws text on the screen
2. NVDA intercepts this information and stores it in a buffer
3. When using screen review to navigate the window, NVDA therefore returns buffered info
4. When NVDA is restarted, the buffered content is lost. As long as the window isn't redroan, NVDA won't be able to access the screen using screen review.

It might be possible to fix this by forcefully redrawing the foreground window and all of its subwindows when NVDA is started. I think it is pretty easy to test this hypothesis.

**Comment by cary-rowen** [comment]
Hi @LeonarddeR 
Do you have any new progress on this?

**Comment by LeonarddeR** [comment]
NO. Note that I have never worked on this and it is honestly not my intention to do so.

**Comment by Adriani90** [comment]
cc: @CyrilleB79 https://github.com/nvaccess/nvda/issues/7900#issuecomment-488584095 might be of interest to you as well.
